tests/unit/drivers/rank/aggregate/test_aggregate_matches_rank_driver.py

```diff
@@ -27,8 +31,7 @@ def exec_fn(self):
 
 class MockLengthRanker(Chunk2DocRanker):
     def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        self.required_keys = {'length'}
+        super().__init__(query_required_keys=('length', ), match_required_keys=('length', ), *args, **kwargs)
```

Comment:
Maybe add a test with more than one required key?

Reply:
any idea? the interpretability of this test would be lost

Reply:
What about adding `discount` and `price`? We can set ranking score by `discount * price`.

```
```

Reply:
Great idea @nan-wang